inventory/50-kolla

```diff
@@ -120,6 +120,9 @@ control
 [prometheus:children]
 monitoring
 
+[outward-rabbitmq:children]
```

Comment:
For alphabetic ordering, this would go in front of panko. For thematic ordering, maybe better after the "main" rabbitmq entry?